config.proto

```diff
@@ -0,0 +1,75 @@
+syntax = "proto3";
+
+package release;
+
+service Release {
+    // Get information about a release from its id
+    rpc GetRelease(string) returns (Release) {};
+    // List all releases matching the request
+    rpc ListReleases (ListReleasesRequest) returns (ListReleasesReponse) {};
+}
+
+message ListReleasesRequest {
+    // List only releases created after, if != 0
+    int64 created_after = 1;
+    // 
+}
+
+enum Status {
+    // Default.
+    UNUSED_STATUS = 0;
+    // The Release is in progress.
+    PENDING = 1;
+    // The Release is succesfull.
+    COMPLETE = 2;
+    // The Release could not complete and needs manual intervension.
+    SUSPENDED = 3;
+    // The Release has been abandoned.
+    ABANDONED = 4;
+}
+
+enum Type {
+    // Default
+    UNSET_TYPE = 0;
+    // Daily release
+    DAILY_RELEASE = 1;
+    // Weekly release
+    WEEKLY_RELEASE = 2;
+    // Public Release
+    OFFICIAL_RELEASE = 3;
```

Comment:
Now that we have releases coming from release branches... I suspect the type should come from the DAG/airflow directly as string labels, and then the UI should handle/group them accordingly, instead of defining some fixed enums.

Reply:
done